include/NovelRT/Maths/Utilities.h

```diff
@@ -22,17 +22,30 @@ namespace NovelRT::Maths
     class Utilities
     {
     public:
+        /**
+         * @brief Returns the de Bruijn Sequences to Index a 1 in a Word
+         * @details To learn more about the de Bruijn Sequences, refer to this documentation: http://supertech.csail.mit.edu/papers/debruijn.pdf
+         * @returns A 32 items array of 8 bits integer containing the de Bruijn Sequences
+         */
```

Comment:
I'm not sure if this suggestion is correct, as I was struggling to understand the wording here in the original. Sorry. :(
```suggestion
        /**
         * @brief Returns the De Bruijn Sequence for indexing a Word.
         * @details To learn more about the de Bruijn Sequences, refer to this documentation: http://supertech.csail.mit.edu/papers/debruijn.pdf
         * @returns A 32 items array of 8 bits integer containing the de Bruijn Sequence.
         */
```